ale_linters/elixir/credo.vim

```diff
@@ -3,8 +3,8 @@
 function! ale_linters#elixir#credo#Handle(buffer, lines) abort
   " Matches patterns line the following:
   "
-  " stdin:19: F: Pipe chain should start with a raw value.
-  let l:pattern = '\v^stdin:(\d+):?(\d+)?: (.): (.+)$'
+  " lib/filename.ex:19:7: F: Pipe chain should start with a raw value.
+  let l:pattern = '\v^.+:(\d+):?(\d+)?: (.): (.+)$'
```

Comment:
I'll accept this, but you might want to try a few Windows filenames including the drive letter. These have caused some problems before.